deploy/central/tasks-chart/templates/task-ods-build-typescript-with-sidecar.yaml

```diff
@@ -72,6 +72,15 @@ spec:
     description: 'Node.js version to use - supported versions: 16'
     name: node-version
     type: string
+  - default: dist
+    description: The directory into which `npm run build` places files. The files
```

Comment:
What do you think about the wording "Must match the directory into which `npm run build` places files."?